pandas/core/frame.py

```diff
@@ -11083,6 +11083,25 @@ def _get_data() -> DataFrame:
                 ).iloc[:0]
                 result.index = df.index
                 return result
+
+            if df.shape[1] and name != "kurt":
```

Comment:
IIUC kurt is excluded here bc GroupBy doesnt support it.  Can you comment to that effect

Reply:
added a comment here